class-rest-api.php

```diff
@@ -262,15 +263,23 @@ public function get_item_schema() {
 	/**
 	 * Convert a Mapping object to data we can return
 	 *
-	 * @param Mapping $mapping
+	 * @param Mapping         $mapping
+	 * @param WP_REST_Request $request
 	 * @return array
 	 */
-	protected function mapping_to_data( $mapping ) {
-		return array(
+	protected function mapping_to_data( $mapping, $request ) {
```

Comment:
`prepare_item_for_response` is the typical name we have for this method, not required of course